independent-publisher-connectors/Zoom Meetings/apiDefinition.swagger.json

```diff
@@ -0,0 +1,141 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zoom Meetings",
+    "description": "This Connector will help to automate Zoom Meeting Operations",
+    "version": "1.0"
+  },
+  "host": "api.zoom.us",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v2/users/me/meetings": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}

```

Comment:
@A-Deekshith Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid response schema definition for all the operations in the swagger.